addons/mrp/mrp_view.xml

```diff
@@ -562,6 +562,52 @@
                 </search>
             </field>
         </record>
+
+        <record id="product_template_search_view_procurment" model="ir.ui.view">
+             <field name="name">product.template.search.bom</field>
+             <field name="model">product.template</field>
+             <field name="inherit_id" ref="product.product_template_search_view"/>
+             <field name="arch" type="xml">
+                 <xpath expr="//filter[@name='products']" position="after">
+                     <separator/>
+                    <filter string="Components" name="components" icon="terp-accessories-archiver" domain="[('bom_ids','not in',[])]"/>
+                 </xpath>
+             </field>
+        </record>
+
+        <record id="view_mrp_product_template_form_inherited" model="ir.ui.view">
+            <field name="name">product.form.mrp.inherited</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="stock.view_template_property_form"/>
+            <field name="arch" type="xml">
+                <group name="sale_condition" position="inside">
+                    <label for="produce_delay" attrs="{'invisible':[('type','=','service')]}"/>
+                    <div attrs="{'invisible':[('type','=','service')]}">
+                        <field name="produce_delay" class="oe_inline"/> days
+                    </div>
+                </group>
+            </field>
+        </record>
+        
+        <record id="product_template_action" model="ir.actions.act_window">
```

Comment:
@tde-banana-odoo ,
Yes, This is duplicate view. but, do not need this then I remove it.